Add a note to XDR readme (#27899)
demisto/content: Packs/CortexXDR/Integrations/CortexXDRIR/README.md

* `Owner` and `closeReason` mappings are done using the integration code, therefore they are not part of the out-of-the-box mapper and should not be specified in any future mapper.

### Fetch Behavior vs Mirroring

> Note: All incidents, including those with a "resolved" status, will be fetched into XSOAR as "active" incidents to enable the execution of our automations. However, the original resolved status of the incidents will be preserved in the incident details. If the user prefers to keep certain incidents closed, they can utilize the "Incident Statuses to Fetch" filter during the configuration stage and choose not to import those specific incidents. Alternatively, they can utilize pre-processing rules to define specific types of incidents to be imported as closed.

Reviewer: ```suggestion
Note: All incidents, including those with a "resolved" status, will be fetched into Cortex XSOAR as "active" incidents to enable the execution of our automations. However, the original resolved status of the incidents will be preserved in the incident details. If you prefer to keep certain incidents closed, you can utilize the "Incident Statuses to Fetch" filter during the configuration stage and choose not to import those specific incidents. Alternatively, you can utilize pre-processing rules to define specific types of incidents to be imported as closed.
```

> Regarding mirroring, if you have already imported an incident and the mirroring feature is enabled, changing the incident's status to resolved on the XDR platform will trigger the mirroring process, resulting in the closure of the incident in XSOAR.

Reviewer: ```suggestion
Regarding mirroring, if you have already imported an incident and the mirroring feature is enabled, changing the incident's status to resolved on the Cortex XDR platform will trigger the mirroring process, resulting in the closure of the incident in Cortex XSOAR.
```
